content/shared/influxdb3-internals-reference/_index.md

```diff
@@ -0,0 +1,3 @@
+### [Data Durability](./durability.md)
+
+InfluxDB 3 ensures data durability through a structured write path that balances performance, persistence, and query efficiency. Data moves through validation, in-memory buffering, Write-Ahead Log (WAL) persistence, and long-term storage in Parquet format to provide reliability and optimized access.
```

Comment:
```suggestion
{{< children >}}
```
Hugo considers `_index.md` as the root metadata page for it's folder (section), so we can use the shortcut to generate the list of pages within the section.